readme.md

```diff
@@ -23,6 +23,8 @@ Add `"dev": "webpack-watch-server"` to your package.json scripts.
 
 Run `npm run dev` to start Webpack in watch mode. The server automatically starts and restarts after each build.
 
+Use `webpack-watch-server --config my-custom-config.js` to specify an alternative webpack config.
```

Comment:
This addition should follow the first line under "Usage", and could incorporate the phrasing `to specify a custom Webpack config file path`. Is it possible to use `--config subdirectory/my-custom-config.js`?